### DISCUSSION TITLE
New cache control parameter descriptions

### DISCUSSION THREAD

**Discussion by jsirianni** [goreleaser/goreleaser#4610] [Ideas, Issue Triage, and General discussions]
Goreleaser 1.24 has a new cache control section for blob storage (thank you!)

```yaml
    cache_control:
      - max-age=9999
      - public
```

Is max-age in seconds?

What is "public" for?

I am using Google Cloud Storage, if it matters.

Thanks!

**Comment by caarlos0** [comment]
It this the cache-control http header, as in here https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

**Comment by jsirianni** [comment]
Thanks!
